dist/tools/jlink/jlink.sh

```diff
@@ -122,11 +122,20 @@ test_tui() {
     fi
 }
 
+test_opts() {
```

Comment:
I had this before. I can rename it back. Shall I then also change `JLINK_OPTS` to `JLINK_SERIAL`?